java-frontend/src/main/java/org/sonar/java/DefaultJavaResourceLocator.java

```diff
@@ -20,38 +20,29 @@
 package org.sonar.java;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Preconditions;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSortedSet;
 import com.google.common.collect.Maps;
 import java.io.File;
 import java.util.Collection;
 import java.util.Map;
 import org.sonar.api.batch.fs.InputFile;
-import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
 import org.sonar.plugins.java.api.JavaFileScannerContext;
 import org.sonar.plugins.java.api.JavaResourceLocator;
 
 public class DefaultJavaResourceLocator implements JavaResourceLocator {
 
-  private static final Logger LOG = Loggers.get(JavaResourceLocator.class);
+  private static final Logger LOG = Loggers.get(DefaultJavaResourceLocator.class);
```

Comment:
I was thinking to create a rule about this :)

Reply:
https://jira.sonarsource.com/browse/SONARJAVA-2781